rotkehlchen/db/accounting_rules.py

```diff
@@ -204,6 +205,33 @@ def add_linked_setting(
                 f'Link of rule for {rule_property} to setting {setting_name} already exists',
             ) from e
 
+    def get_valid_entry_combinations(
```

Comment:
don't do it like this. Instead what you want to learn is if a set of entries have valid rules or not. PAss the entries here and do a direct DB check.